sdk/keyvault/azure-security-keyvault-keys/src/samples/java/com/azure/security/keyvault/keys/cryptography/CryptographyAsyncClientJavaDocCodeSnippets.java

```diff
@@ -102,10 +104,11 @@ public CryptographyAsyncClient createAsyncClientWithPipeline() {
      */
     public void getKeySnippets() {
         CryptographyAsyncClient cryptographyAsyncClient = createAsyncClient();
+
         // BEGIN: com.azure.security.keyvault.keys.cryptography.CryptographyAsyncClient.getKey
         cryptographyAsyncClient.getKey()
-            .subscriberContext(reactor.util.context.Context.of(key1, value1, key2, value2))
```

Comment:
Use `contextWrite()` instead.
https://projectreactor.io/docs/core/release/reference/#context

Reply:
Should all async samples include the context?